services/slack/README.md

```diff
@@ -3,26 +3,96 @@ Slack
 
 [![Image of Sack][producti]][product]
 
-Our team communication platform because we like to talk to each other.
+We use Slack to stay in touch with each other no matter where we are physically. It’s a great tool, and the following outlines useful tips to keep Slack useful—and navigable–for everyone.
 
 [product]: https://slack.com/
 [producti]: http://i.imgur.com/0u1iR8W.png
 
 ## Channels
 
-Slack allows us to create channels to talk to everyone at once, think chat room.
+Slack allows us to create channels to talk to everyone at once, think "chat room".
 
-### General & Random
+### #general & #random
 
-By default slack has two channels, General and Random, and all users are added. General is a place to talk about non work related stuff and Random is a scary place of pure randomness.
+By default slack has two channels, #general and #random, and all users are added. General is a place to talk about work related stuff and Random is a scary place of pure randomness.
+
+### Sparkbox channels
+Official "Sparkbox Sanctioned" channels begin with a `-`. See the note on prefixes below.
+
+##### #-awesomeness
+A channel for sharing awesome notes about coworkers.
+
+##### #-biz
+A place for **bizBot** to keep Sparkbox informed about business development.
+
+##### #-conferences
+The channel for sharing interesting conferences, Call for Speakers, and planning the conference you'd like to attend.
+
+##### #-dayton & #-pittsburgh
+Channels to discuss office-specific happenings, such as lunch plans.
+
+##### #-gems
+A collection of quotations from your Sparkbox community, currated in part by **gem-me-bot**. Share a gem publicly with the `:earth-americas:` reaction, or hide it from the public eye with the `:x:` response.
```

Comment:
could like here for more info: https://seesparkbox.com/foundry/sparkbox_gems

Reply:
Ohhh nice!